docs/using/flowpipe/mods.md

```diff
@@ -70,6 +70,26 @@ applicable (if left empty, will default to `*`). When finished, click **Install
 
 After the mod is installed, you will be prompted to [set its variables](#managing-mod-variables).
 
+## Installing an Archived Mod
+
+You can install an archived mod by selecting **Install from mod archive** option. This will allow you to upload your own archived mod file subject to the following rules:
+
+- The mod archive must be one of the following formats: `.tar.gz`, `.tar`, `.zip`.
+- The archive must contain a `mod.pp` or `mod.sp` file at the root location and it should not be nested inside any folder.
+- Size of the archive should not exceed 20MB.
+
+Best practices to follow when creating an archived mod file:
+
+- Archive the mod file by navigating to the directory containing the `mod.pp` or `mod.sp` file to prevent the mod file from being nested inside any folder.
+- Ignore any hidden / unnecessary files or directories that are not required for the mod. In that essence we recommend using one of the following scripts based on the type of archive you are creating:
+  - `.tar.gz` - `tar -czf "$(basename "$PWD").tar.gz" --exclude=".*" --exclude="*/.*" *`
+  - `.tar` - `tar -cf "$(basename "$PWD").tar" --exclude=".*" --exclude="*/.*" *`
```

Comment:
Is there a reason why you would do an uncompressed tar ball?   Should we only show 2 options - .zip and .tar.gz ?